charts/timber-observation-service/templates/deployment.yaml

```diff
@@ -0,0 +1,119 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "observation-service.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "observation-service.labels" . | nindent 4 }}
+spec:
+  {{- if not .Values.observationService.autoscaling.enabled }}
+  replicas: {{ .Values.observationService.replicaCount }}
+  {{- end }}
+  selector:
+    matchLabels:
+      app: {{ .Release.Name }}
+      release: {{ .Release.Name }}
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 0
+  template:
+    metadata:
+      labels:
+        app: {{ .Release.Name }}
+        release: {{ .Release.Name }}
```

Comment:
I think we can remove this since `release: {{ .Release.Name }}` is already part of the common labels in `_helpers.tpl`